src/rust/e2e-tests/src/test_utils/context.rs

```diff
@@ -0,0 +1,171 @@
+use bytes::Bytes;
```

Comment:
# A note:
everything in `src/test_utils` here is just moved, no *new* functionality except the SetupGeneratorOptions refactor (previously multiple arguments)